sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -216,16 +216,16 @@ object SQLConf {
         "for using switch statements in InSet must be non-negative and less than or equal to 600")
       .createWithDefault(400)
 
-  val OPTIMIZER_LIKE_ALL_CONVERSION_THRESHOLD =
-    buildConf("spark.sql.optimizer.likeAllConversionThreshold")
+  val OPTIMIZER_MULTI_LIKE_CONVERSION_THRESHOLD =
+    buildConf("spark.sql.optimizer.multiLikeConversionThreshold")
       .internal()
-      .doc("Configure the maximum size of the pattern sequence in like all. Spark will convert " +
-        "the logical combination of like to avoid StackOverflowError. 200 is an empirical value " +
-        "that will not cause StackOverflowError.")
+      .doc("Configure the maximum size of the pattern sequence in like [all|any|some]. Spark " +
```

Comment:
Shall we capitalize: `like [all|any|some]` ->`LIKE [ALL|ANY|SOME]`?

Reply:
OK